docs/guide/best_practices.md

```diff
@@ -151,3 +151,18 @@ ezpublish:
 This setting **does not change Symfony behavior** regarding redirection after login. If set, it will only substitute the value set for `default_target_path`. It is therefore still possible to specify a custom target path using a dedicated form parameter.
 
 **Order of precedence is not modified.**
+
+### `anonymous_user_id`
+When a user that is not logged in is accessing an eZ Platform installation, the request will be handled with the permissions of the anonymous user. You can select any user on your system to represent the anonymous user.
+For instance, you can create also separated anonymous user group per siteaccess. Then, you can create the new user, assign him to the new group and make all permissions depend on siteaccess.
```

Comment:
`For instance, you can also create a separate anonymous user group per siteaccess. Then, you can create the new user, assign them to the new group and make all permissions depend on siteaccess.`